.travis.yml

```diff
@@ -25,4 +26,4 @@ script:
   - golint ./...
   - go build .
   - go test -v ./...
-  - ./git-validation -run DCO,short-subject,dangling-whitespace -v
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./git-validation -run DCO,short-subject,dangling-whitespace -v ; fi' 
```

Comment:
I think a better fix would be to build this logic into `git-validation`.  If it can't handle non-PR tests, we don't want to have to copy this guard into every consuming project's invocation.